app/decorators/models/solidus_braintree/spree/credit_card_decorator.rb

```diff
@@ -1,3 +1,2 @@
 Spree::CreditCard.include SolidusBraintree::SkipRequireCardNumbersConcern
-Spree::CreditCard.include SolidusBraintree::AddNameValidationConcern
```

Comment:
Why is this required to support Solidus 3.4?

Reply:
Readded https://github.com/solidusio/solidus_braintree/pull/147/commits/7737d54e666791f078f1cd039f6f1414dba54a9c